packages/search-ui/src/__tests__/actions/addFilter.test.js

```diff
@@ -64,40 +66,57 @@ describe("#addFilter", () => {
 
   it("Adds a new filter", () => {
     expect(
-      subject("test", "value", undefined, {
-        initialFilters: [{ field: "initial", values: ["value"], type: "all" }]
+      subject("test", "value", "any", undefined, {
+        initialFilters: [
+          { field: "initial", values: ["value"], outerType: "all", type: "any" }
+        ]
       }).filters
     ).toEqual([
-      { field: "initial", values: ["value"], type: "all" },
-      { field: "test", values: ["value"], type: "all" }
+      { field: "initial", values: ["value"], outerType: "all", type: "any" },
```

Comment:
Is there still a test that tests when outerType is not used? I think it may be better to keep this test unchanged and add a new test which includes outerType